tfx/experimental/templates/taxi/kubeflow_dag_runner.py

```diff
@@ -71,6 +72,10 @@ def run():
   tfx_image = os.environ.get('KUBEFLOW_TFX_IMAGE', None)
 
   runner_config = kubeflow_dag_runner.KubeflowDagRunnerConfig(
+      # TODO(StubExecutor): Uncomment below to use stub executors.
```

Comment:
How about adding a link to the document above? I think that this would be their first encounter with stub executors for most users.

Reply:
There is an internal lint rule that we cannot use TODOs with this style. Could you change TODOs into `TODO(stub_executor)` format?